.github/workflows/super-linter.yaml

```diff
@@ -0,0 +1,43 @@
+---
+name: All in One Linter
+
+on:
+  push:
+    tags:
+      - v*
+    branches:
+      - master
+      - release-*
+  pull_request:
+    branches:
+      - master
+      - release-*
+
+jobs:
+  Lint:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout Code
+        uses: actions/checkout@v2
+        with:
+          # Full git history is needed to get a proper list of changed files within `super-linter`
+          fetch-depth: 0
+
+      - name: symlink
+        run: ln -sf .yamllint.yml .github/linters/.yaml-lint.yml
+
+      - name: Lint Code Base
+        uses: github/super-linter@v4
```

Comment:
No bash linter?

Reply:
this contains all the linter(most of them) I have not enabled it in this now currently, but will do later